src/actions.js

```diff
@@ -0,0 +1,64 @@
+import C from 'src/constants';
+
+export const startSearch = (phrase, type) => {
+  return {
+    type: C.START_SEARCH,
+    payload: {
+      searchPhrase: phrase,
+      searchType: type,
+      replaceCurrentResultsBy: [],
+      loadingData: true
+    }
+  }
+};
+
+export const resultsToStore = (results) => {
+  return {
+    type: C.RESULTS_TO_STORE,
+    payload: {
+      foundResults: results,
+      loadingData: false
+    }
+  }
+};
```

Comment:
Could be simplified along with other actions  
`export const resultsToStore = results => ({  
  type: C.RESULTS_TO_STORE, 
  payload: {  
    foundResults: results,  
    loadingData: false,  
  },  
});`

Reply:
Ok, this code is refactored